src/core/Tabs/index.tsx

```diff
@@ -1,67 +1,107 @@
 import { css } from "@emotion/css";
 import styled from "@emotion/styled";
 import {
-  StylesProvider,
   Tab as RawTab,
   Tabs as RawTabs,
   TabsProps as RawTabsProps,
+  useTheme,
 } from "@material-ui/core";
-import React from "react";
-import { defaultThemeOptions } from "../styles";
+import React, { ChangeEvent, FormEvent } from "react";
+import { AppThemeOptions } from "../styles/common/defaultTheme";
+import {
+  getColors,
```

Comment:
Update Tabs to use the theme passed to Emotion instead of the hardcoded default theme too. This way if the product's theme changes, the Tabs will respect the new theme object's values